zc_events/utils.py

```diff
@@ -0,0 +1,43 @@
+
+def model_to_dict(instance, included_attributes=[]):
+    """Returns a native python dict corresponding to the selected attributes of a model instance.
+
+    Args:
+        instance: model instance to serialize
+        included_attributes: attributes or methods on instance.
+            It should be formatted as [(attr_on_instance, name_in_the_returned_dict), ..., attr_name, ...]
+    """
+
+    def get_value(obj, field):
```

Comment:
I personally prefer not to have functions inside other functions, you can use private methods for this.